src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
+      }}
+    >
       <div
         style={{
-          padding: '8px',
           display: 'flex',
-          flexDirection: 'column',
           gap: '8px',
-          maxWidth: '360px'
+          minWidth: '846px',
+          padding: '8px'
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="street-input"
+            name="lastPermanentStreet"
+            label="Street:"
+            onChange={handleChange}
+            value={formData.lastPermanentStreet}
+          />
+          <TextField
+            id="city-input"
+            name="lastPermanentCity"
+            label="City:"
+            onChange={handleChange}
+            value={formData.lastPermanentCity}
+          />
+          <TextField
+            id="state-input"
+            name="lastPermanentState"
+            label="State:"
+            onChange={handleChange}
+            value={formData.lastPermanentState}
+          />
+          <TextField
+            id="zip-input"
+            name="lastPermanentZIP"
+            label="ZIP Code:"
+            onChange={handleChange}
+            value={formData.lastPermanentZIP}
+          />
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '8px',
+            padding: '8px',
+            width: '40%'
+          }}
+        >
+          <TextField
+            id="months-homeless-input"
+            name="monthsHomeless"
+            label="Months Houseless Past 3 Years:"
```

Comment:
We should discuss the length of labels for this and the following inputs. They should be short enough that they are fully visible to the user.

Reply:
Please tell me what you want each of the label strings changed to and I will do that.

Reply:
I wanted to open this to discussion to avoid telling anyone what to do. I didn't set it out as a requirement for this PR so let's just leave it for a future one.